authorizations/authorization-ranger/src/test/java/org/apache/gravitino/authorization/ranger/integration/test/RangerHiveE2EIT.java

```diff
@@ -122,6 +143,8 @@ public void startIntegrationTest() throws Exception {
 
     generateRangerSparkSecurityXML();
 
+    setEnv(HADOOP_USER_NAME, "test");
```

Comment:
I think `setEnv()` function will change the OS environment variable. Maybe affected other IT case. So I think we should use environment variable in Gradle script.

Reply:
Removed.